js/foam/u2/SearchBorder.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+CLASS({
+  package: 'foam.u2',
+  name: 'SearchBorder',
+  extends: 'foam.u2.View',
+  documentation: 'Wraps a view of a DAO to add filtering based on a search ' +
+      'query. You can supply a $$DOC{ref:".queryParser"}, or use the default ' +
+      '(KEYWORD).',
+
+  imports: [
+    'data',
+  ],
+  exports: [
+    'filtered_ as data',
```

Comment:
I feel like we should export a proxy to filtered_, so that imports don't have to be dynamic.

I don't think you can export filtered_$Proxy, so maybe make another internal property named which is a ProxyDAO to filtered_ and export that.